grammar.js

```diff
@@ -1231,6 +1233,55 @@ module.exports = grammar({
       $._type,
     ),
 
+
+    _interval_definitions: $ => repeat1(
```

Comment:
I think declaring this as a named node e.g. `interval_definitions` would be nice to have it show up in the syntax tree.